electron-app/src/utils.ts

```diff
@@ -200,11 +202,23 @@ export const getBaseFolder = () => {
   return baseFolder;
 };
 
+export const getBaseFolderReindex = () => {
```

Comment:
This method does the same as getBaseFolder. Maybe we can call it, then just do a path.join(base, '..') to go down one folder

Reply:
There are several conditions in getBaseFolder and due to that in the current scenario, it will make the code complex.